pkg/jsonpath/filter.go

```diff
@@ -241,7 +242,7 @@ func (e basicExpr) ToString() string {
 	return ""
 }
 
-// literal literal = number /
+// literal literal-value = number /
```

Comment:
This change is to disambiguate the two things, the first, appears to be a golang struct name, and the second appears to be the name for the type to be used in other RHS locations.